assets/api-swagger/swagger.json

```diff
@@ -802,7 +802,7 @@
             "name": "id",
             "in": "path",
             "description": "The id of the profile to merge.",
-            "required": false,
+            "required":true,
```

Comment:
Hmm, this does not really match the code [here](https://github.com/Ometria/ometria.api.profiles/blob/efa30dc66a734653a86e8e53e6783ea604ac390a/profiles/merge_profile.py#L46) - the implementation has both, id and email_address, optional (technically), and only if both are missing, the API call returns with an appropriate error message.

Reply:
Ah ok, I've got this the wrong way round, new commit below.